bifrost_jupyter_extension/bifrost.py

```diff
@@ -39,7 +39,7 @@ class BifrostWidget(DOMWidget):
     current_dataframe_index = Int(0).tag(sync=True)
     query_spec = Dict({}).tag(sync=True)
     graph_spec = Dict({}).tag(sync=True)
-    flags = Dict({}).tag(sync=True)
+    args = Dict({}).tag(sync=True)
```

Comment:
Super minor, but it might be easier to understand if this is called something like `plot_function_args`